### DISCUSSION TITLE
spawn low level bots

### DISCUSSION THREAD

**Discussion by Nicolas-Puccio** [trickerer/Trinity-Bots#566] [Q&A] [answered]
i disabled wandering bots as i am not interested in them, but i want to spawn a few lvl 1 bots to lvl up with
edit: nvm i just read the readme it should be set to my lvl, but how can i control their level? can i set them to be lower lvl than me?
edit2: or force their damage to be lower, i tried setting

NpcBot.Mult.Damage.Levels = 0.1,0.1,0.1,0.1,0.1,0.1,0.1,0.1,0.1
NpcBot.Mult.Damage.Warrior           = 0.1,etc

**Comment by trickerer** [comment, marked as answer]
You can't adjust bots' level compared to player's currently, they'are always same level as you except for few extra classes which have permanent level bonus.

Settings you mention are indeed used to adjust bots' damage, also `NpcBot.Mult.Damage.Physical / NpcBot.Mult.Damage.Spell`. Don't forget to reload the config (`.npcbot reload`) or restart the server
